client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -90,6 +92,25 @@ const UserPage = createClass({
 		  </td>;
 	},
 
+	handleFilterTextChange : function(e){
+		this.setState({
+			filterString : e.target.value
+		});
+		return;
+	},
+
+	renderFilterOption : function(){
+		return <td>
+			<label>
+				Filter Title:
```

Comment:
Change to just "Filter:", or maybe use fa-search icon? https://fontawesome.com/v4.7/icon/search